setup/resource_calendar_leaves_province/odoo/addons/resource_calendar_leaves_province

```diff
@@ -0,0 +1 @@
+../../../../resource_calendar_leaves_province
```

Comment:
Please rename the module to `resource_calendar_leaves_province`.

Reply:
After a 2nd thought, maybe `resource_leaves_geographic` is better. I guess that it'd make sense if the same module can be extended later for leaves by country (in case of a multinational company). Well, I let you choose between these.